docs/source/examples/capacity_expansion_tutorial.ipynb

```diff
@@ -54,9 +54,11 @@
     "\n",
     "# set the solver based on operating system -- assumes glpk or cbc is installed.\n",
     "if \"win32\" in sys.platform:\n",
-    "    solver = 'cplex'\n",
+    "    solver = 'cplex_direct'\n",
     "elif \"linux\" in sys.platform:\n",
     "    solver = \"cbc\"\n",
+    "elif \"darwin\" in sys.platform:\n",
+    "    solver = \"cplex_direct\"\n",
```

Comment:
Do you know this will always be necessary or does it depend on how you installed CPLEX?